test/natalie/array_test.rb

```diff
@@ -475,6 +475,15 @@
     end
   end
 
+  describe '#collect' do
+      it 'returns a new array of the result of evaluating the block for each item in the array' do
+        result = [1, 2, 3].collect do |i|
+          i * 2
+        end
+        result.should == [2, 4, 6]
+      end
+    end
+
```

Comment:
This is just a copy and paste of the `#map` test is there some easier way to test aliases?
Since now you would have to double test writing as well (or you might just forget something has an alias

Reply:
I supposed you could do something like this:

```ruby
['map', 'collect'].each do |method_name|
  describe "##{method_name}" do
    it 'whatever' do
      result = [1, 2, 3].send(method_name) do |i|
        i * 2
      end
      result.should == [2, 4, 6]
    end
  end
end
```

BUT I'm not sure that's better. I think I'd rather just have the duplication. :man_shrugging:

Reply:
Alright, kept it like this